testplugins/src/main/java/org/spongepowered/test/MultiWorldTest.java

```diff
@@ -40,22 +40,38 @@
 
     @Override
     public void disable(CommandSource src) {
-        Sponge.getServer().getWorld("temp").ifPresent(world -> Sponge.getServer().unloadWorld(world));
+        Sponge.getServer().getWorld("no-save").ifPresent(world -> {
```

Comment:
Pull out the `Sponge.getServer()` as a local variable at this point with how many times it's being called.